### ISSUE TITLE
Open links in the default browser

### ISSUE THREAD

**Issue by kudlav** [ThatOneCalculator/google-chat-tauri#6]
Hi,
received links are not clickable simply by the left click.
Links can be opened by the middle button or by the right click -> Open link in a new window.

But the links are opened in a new Google Chat Tauri internal browser window (where I'm not logged in to anything).
I'd expect to open the links in the default browser.

![obrazek](https://github.com/ThatOneCalculator/google-chat-tauri/assets/11516259/e4265472-dca8-4fcb-a0fe-083ee79a24a3)

Google Chat Tauri v0.0.7
Win 11 23H2

**Comment by dax702** [comment]
Yes, I just discovered this too, bummer as I liked everything else about it

**Comment by ThatOneCalculator** [comment]
Unfortunately, I don't think there's anything I can do... there is [this](https://v2.tauri.app/reference/javascript/shell/#open) but that has to be called manually -- I can't listen to anchor tag clicks afaik.
